src/services/authServices.js

```diff
@@ -57,3 +58,32 @@ export const logoutService = async (token) => {
     throw err;
   }
 };
+
+export const getUserProfileService = async(id) => {
```

Comment:
this function does the same thing as the `findUserById`. I think it's redundant.

Reply:
There is a `delete` line in this one